repository/src/main/java/org/apache/atlas/discovery/EntityLineageService.java

```diff
@@ -863,13 +863,24 @@ private boolean isVertexActive(AtlasVertex vertex) {
     }
 
     private List<Pair<AtlasEdge, String>> getUnvisitedProcessEdgesWithOutputVertexIds(boolean isInput, AtlasLineageContext lineageContext, Set<Pair<String, String>> paginationCalculatedProcessOutputPair, AtlasVertex processVertex) {
-        return getEdgesOfProcess(isInput, lineageContext, processVertex)
-                .stream()
-                .map(processEdge -> Pair.of(processEdge, processEdge.getInVertex()))
-                .filter(pair -> pair.getRight() != null)
-                .map(pair -> Pair.of(pair.getLeft(), pair.getRight().getIdForDisplay()))
-                .filter(pair -> !paginationCalculatedProcessOutputPair.contains(Pair.of(getGuid(processVertex), pair.getRight())))
-                .collect(Collectors.toList());
+        List<Pair<AtlasEdge, String>> unvisitedProcessEdgesWithOutputVertexIds = new ArrayList<>();
```

Comment:
We were checking for lineageContext.getIgnoredProcesses() in getEdgesOfProcess method, now that is removing where are we checking that?

Reply:
just added, thank you